code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -1753,6 +1753,13 @@
 	name = "farwa cube"
 	monkey_type =/mob/living/carbon/monkey/tajara
 
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wolpincube
+	name = "wolpin cube"
+	monkey_type = /mob/living/carbon/monkey/vulpkanin
+/obj/item/weapon/reagent_containers/food/snacks/monkeycube/wrapped/wolpincube
+	name = "wolpin cube"
+	monkey_type = /mob/living/carbon/monkey/vulpkanin

```

Comment:
honestly I'd do away with the wrapped feature entirely, it's pointless busywork really